### PR TITLE
docker push

### PR DESCRIPTION
**Pull request by gitadvisor**
- [x] Mark this if you consider it ready to merge
- [x] I've added tests (optional)
- [x] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:14.04
```
Comment (trufae) [inline Dockerfile:1]: ubuntu14 srsly?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  apt-get install -y build-essential && \
+  apt-get install -y software-properties-common && \
+  apt-get install -y byobu curl git htop man unzip vim wget && \
+  rm -rf /var/lib/apt/lists/*
+
+
+# Define working directory.
+WORKDIR /.tmp
+
+RUN git clone https://github.com/gitadvisor/radare2 && \
```
Comment (trufae) [inline Dockerfile:17]: uh? wrong repo

**Comment by trufae** [comment]
We already have docker publish in a separate protected repo. docker credentials wont be shipped here. also the current docker ships all the plugins and its configurable and uses modern ubuntu. and its build for all the available archs and integrated with snap.

File: Dockerfile (hunk 1 above)
Comment (gitadvisor) [reply Dockerfile:1]: ubuntu:14.04

File: Dockerfile (hunk 2 above)
Comment (gitadvisor) [reply Dockerfile:17]: https://github.com/radareorg/radare2

**Pull request closed without merging**